### PR TITLE
CTW-595/patient history standalone

### PR DESCRIPTION
**Pull request by WesleyKapow**
CTW-595

Add `usePatientHistory` hook to open request drawer. Hook currently only provides the `openHistoryRequestDrawer` function. Down the road we can push more content into this, such as last request, in progress info, etc.

New story showing off the hook via a custom link to invoke the `openHistoryRequestDrawer()` helper.
Screenshot 2023-02-01 at 10 18 54 AM

Updated form field error color AND made form fields show generic message in footer:
Screenshot 2023-02-01 at 10 12 23 AM

### CODE REVIEW

File: .changeset/new-books-impress.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@zus-health/ctw-component-library": patch
```
Comment (pmadria3) [inline .changeset/new-books-impress.md:2]: nit: should this be minor? Not that it really makes too much of a difference

File: src/components/content/patient-history/story-helpers/mocks/requests.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+);
+
+// https://api.dev.zusapi.com/patient-history/patient/u12345/refresh?consent=1
+export const mockPatientHistoryPost = rest.post(
+  "https://api.dev.zusapi.com/patient-history/patient/:patientId/refresh",
+  (req, res, ctx) => {
+    // eslint-disable-next-line no-console
+    console.log("Sent Patient History Request:");
+    // eslint-disable-next-line no-console
+    console.log(req.headers.all());
```
Comment (pmadria3) [inline src/components/content/patient-history/story-helpers/mocks/requests.ts:26]: Is there a reason to console these?

**Review by pmadria3 — APPROVED** [review]
LGTM -> left couple minor things

File: .changeset/new-books-impress.md (hunk 1 above)
Comment (WesleyKapow) [reply .changeset/new-books-impress.md:2]: Eh maybe? I guess adding new features should be? I'm gonna not worry about it here.

File: src/components/content/patient-history/story-helpers/mocks/requests.ts (hunk 2 above)
Comment (WesleyKapow) [reply src/components/content/patient-history/story-helpers/mocks/requests.ts:26]: Just as feedback for users of storybook.. so we can at least see some type of request info in the console.

**Pull request merged**